content/feature/Balance Sheet Implementation - Hellhound14.md

```diff
@@ -0,0 +1,40 @@
+|Date|Profit ($)|Loss ($)|Monthly Earnings ($)|Total ($)|
+|----|----------|--------|--------------------|---------|
+|"2018-01-28"|576|543|33|33|
+|"2018-02-28"|675|521|154|187|
+|"2018-03-28"|432|653|-221|-34|
+|"2018-04-28"|765|321|444|410|
+
+
+### Description
+Creating A Table On A SQL Server Can Help You. `How You Ask?` Well….

```

Comment:
For descriptions in this gallery, we want to avoid using first person in the paragraph.

Also, try to keep the wording professional, and capitalized, sentence wise, not word wise.